app/src/main/java/com/example/habitsmasher/ui/dashboard/DashboardFragment.java

```diff
@@ -1,46 +1,80 @@
 package com.example.habitsmasher.ui.dashboard;
 
+import android.content.Context;
 import android.os.Bundle;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-import android.widget.TextView;
 
 import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
-import androidx.lifecycle.Observer;
-import androidx.lifecycle.ViewModelProvider;
+import androidx.recyclerview.widget.DefaultItemAnimator;
+import androidx.recyclerview.widget.ItemTouchHelper;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
 
+import com.example.habitsmasher.Habit;
+import com.example.habitsmasher.HabitList;
 import com.example.habitsmasher.R;
-import com.example.habitsmasher.databinding.FragmentDashboardBinding;
 
-public class DashboardFragment extends Fragment {
+import java.util.ArrayList;
 
-    private DashboardViewModel dashboardViewModel;
-    private FragmentDashboardBinding binding;
+public class DashboardFragment extends Fragment {
+    private final HabitList _habitList = new HabitList();
+    private final ArrayList<Habit> _habits = _habitList.getHabitList();
+    private HabitItemAdapter _habitItemAdapter;
 
+    @Override
     public View onCreateView(@NonNull LayoutInflater inflater,
                              ViewGroup container, Bundle savedInstanceState) {
-        dashboardViewModel =
-                new ViewModelProvider(this).get(DashboardViewModel.class);
-
-        binding = FragmentDashboardBinding.inflate(inflater, container, false);
-        View root = binding.getRoot();
-
-        final TextView textView = binding.textDashboard;
-        dashboardViewModel.getText().observe(getViewLifecycleOwner(), new Observer<String>() {
-            @Override
-            public void onChanged(@Nullable String s) {
-                textView.setText(s);
-            }
-        });
-        return root;
+        Context context = getContext();
+        _habitItemAdapter = new HabitItemAdapter(context, _habits);
+
+        LinearLayoutManager layoutManager = new LinearLayoutManager(context,
+                                                                    LinearLayoutManager.VERTICAL,
+                                                                    false);
+
+        View view = inflater.inflate(R.layout.fragment_dashboard, container, false);
+
+        initializeRecyclerView(layoutManager, view);
+
+        return view;
+    }
+
+    private void initializeRecyclerView(LinearLayoutManager layoutManager, View view) {
+        RecyclerView recyclerView = view.findViewById(R.id.recycler_view_items);
+        recyclerView.setItemAnimator(new DefaultItemAnimator());
+        recyclerView.setLayoutManager(layoutManager);
+        recyclerView.setAdapter(_habitItemAdapter);
+        new ItemTouchHelper(_itemTouchHelperCallback).attachToRecyclerView(recyclerView);
```

Comment:
The **RecyclerView** is initialized here. The following pieces are initialized:
* Item animation
* Layout manager
* Custom adapter
* ItemTouchHelper, which adds the tap/swipe listener functionality